en/core-api/direct-debit.md

```diff
@@ -345,16 +498,15 @@ HTTP POST request with JSON body will be sent to Merchant's **notification url**
 ```
 <!-- tabs:end -->
 
-<div class="my-card">
 
-#### [Handling Webhook HTTP Notification](/en/after-payment/http-notification.md)
-</div>
+## Switching To Production
+To use Midtrans *Production* environment (accept real payment from real customer), please make sure to follow the steps given below.
```

Comment:
(accept real payment from real customer) <- Is this required? Check in other files too,
remove "please make sure to"